.github/workflows/compute.yml

```diff
@@ -36,13 +38,16 @@ jobs:
 
       - uses: oven-sh/setup-bun@v1
 
-      - name: Generate Supabase Types
-        run: |
-          yarn install
-          yarn run "supabase:generate:remote"
-        env:
-          SUPABASE_PROJECT_ID: ${{ secrets.SUPABASE_PROJECT_ID }}
-          SUPABASE_ACCESS_TOKEN: ${{ secrets.SUPABASE_ACCESS_TOKEN }}
+      - name: Install dependencies
+        run: yarn install
+
+      # - name: Generate Supabase Types
```

Comment:
Should be removed.

Reply:
I actually commented these out just so it wouldn't commit types from my db to the repo during this pr. Should I return or remove, sorry?

Reply:
It is probably needed but no in the compute file, so can be removed from here.